signer/src/transaction_signer.rs

```diff
@@ -796,13 +1110,24 @@ where
             return Err(Error::MissingStateMachine);
         };
 
+        let mut frost_coordinator = if let StateMachineId::RotateKey(_, _) = state_machine_id {
+            self.wsts_frost_state_machines.get_mut(&state_machine_id)
+        } else {
+            None
+        };
+
         let outbound_messages = state_machine.process(msg).map_err(Error::Wsts)?;
 
         for outbound_message in outbound_messages.iter() {
             // The WSTS state machine assume we read our own messages
             state_machine
                 .process(outbound_message)
                 .map_err(Error::Wsts)?;
+
+            if let Some(frost_coordinator) = &mut frost_coordinator {
+                let (_, result) = frost_coordinator.process_message(outbound_message)?;
+                tracing::info!(?outbound_message, ?result, state = ?frost_coordinator.state, "\x1b[1;36mfrost coordinator\x1b[0m relay_message result")
```

Comment:
nit: I would remove the colors, and also should we log this only if result is not none?

Reply:
Leftover from debugging -- removed the log altogether [5d2f807](https://github.com/stacks-network/sbtc/pull/1285/commits/5d2f80776a05c2b8ca8ee79f945664ffe451c463)